openmmtools/mcmc.py

```diff
@@ -1119,6 +1119,158 @@ def _get_integrator(self, thermodynamic_state):
+        Accumulate the shadow work performed by the symplectic substeps, in the global `shadow_work`
+    measure_heat : boolean, default: False
+        Accumulate the heat exchanged with the bath in each step, in the global `heat`
+
+    Examples
+    --------
+    First we need to create the thermodynamic state and the sampler
+    state to propagate. Here we create an alanine dipeptide system
+    in vacuum.
+
+    >>> from simtk import unit
+    >>> from openmmtools import testsystems
+    >>> from openmmtools.states import SamplerState, ThermodynamicState
+    >>> test = testsystems.AlanineDipeptideVacuum()
+    >>> sampler_state = SamplerState(positions=test.positions)
+    >>> thermodynamic_state = ThermodynamicState(system=test.system, temperature=298*unit.kelvin)
+
+    Create a Langevin move with default parameters
+
+    >>> move = LangevinSplittingDynamicsMove()
+
+    or create a Langevin move with specified splitting.
+
+    >>> move = LangevinSplittingDynamicsMove(splitting="V { R O R } V")
+
+    Where this splitting is a 5 step symplectic integrator:
+
+        1. Half-step velocity updates (V),
+        2. Hybrid Metropolized step around the half position updates (R) with Ornstein-Uhlenbeck (O) interactions with
+            the stochastic heat bath interactions.
+        3. A final half-step in velocity
+
+    Perform one update of the sampler state. The sampler state is updated
+    with the new state.
+
+    >>> move.apply(thermodynamic_state, sampler_state)
+    >>> np.allclose(sampler_state.positions, test.positions)
+    False
+
+    The same move can be applied to a different state, here an ideal gas.
+
+    >>> test = testsystems.IdealGas()
+    >>> sampler_state = SamplerState(positions=test.positions)
+    >>> thermodynamic_state = ThermodynamicState(system=test.system,
+    ...                                          temperature=298*unit.kelvin)
+    >>> move.apply(thermodynamic_state, sampler_state)
+    >>> np.allclose(sampler_state.positions, test.positions)
+    False
+
+    """
+
+    def __init__(self, timestep=1.0 * unit.femtosecond, collision_rate=10.0 / unit.picoseconds,
+                 n_steps=1000, reassign_velocities=False, splitting="V R O R V", constraint_tolerance=1.0e-8,
+                 measure_shadow_work=False, measure_heat=False, **kwargs):
+        super(LangevinSplittingDynamicsMove, self).__init__(n_steps=n_steps,
+                                                            reassign_velocities=reassign_velocities,
+                                                            timestep=timestep,
+                                                            collision_rate=collision_rate,
+                                                            **kwargs)
+        self.splitting = splitting
+        self.constraint_tolerance = constraint_tolerance
+        self.measure_shadow_work = measure_shadow_work
+        self.measure_heat = measure_heat
```

Comment:
Because the class is actually a subclass of `LangevinDynamicsMove` and not of `BaseIntegratorMove`, calling the super class in `__init__`, `__setstate__` and `__getstate__` manipulates the `collision_rate` and `timestep` properties already.